Components/DatePicker/DatePicker.js

```diff
@@ -48,21 +56,24 @@ export const DatePicker = (props) => {
 
   return (
     <Calendar
-      current={Date()}
+      current={setCurrent()}
```

Comment:
Finally!! Thanks for figuring out this bug its nice the Calendar doesn't always snap to the current date anymore.